serve.go

```diff
@@ -177,6 +178,7 @@ func serve(args []string) error {
 	} else {
 		handler = r
 	}
+	handler = handlers.CombinedLoggingHandler(os.Stdout, handler)
```

Comment:
Yeah, I saw that. Generally stdout = normal, stderr = errors. But for a long-running daemon those conventions are sorta.. I dunno.

Eventually of course the logging framework should be flexible enough to log to file, network, API, blah blah blah.